python/python/tests/tests_server_modules/test_ft.py

```diff
@@ -0,0 +1,105 @@
+from typing import List
+
+import pytest
+from glide.async_commands.server_modules import ft
+from glide.async_commands.server_modules.ft_options.ft_create_options import (
+    DataType,
+    DistanceMetricType,
+    Field,
+    FtCreateOptions,
+    NumericField,
+    TextField,
+    VectorAlgorithm,
+    VectorField,
+    VectorFieldAttributesHnsw,
+    VectorType,
+)
+from glide.config import ProtocolVersion
+from glide.constants import OK
+from glide.glide_client import TGlideClient
+
+
+@pytest.mark.asyncio
+class TestVss:
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_vss_create(self, glide_client: TGlideClient):
+        fields: List[Field] = []
+        field1: TextField = TextField("title")
+        field2: NumericField = NumericField("published_at")
+        field3: TextField = TextField("category")
+        fields.append(field1)
+        fields.append(field2)
+        fields.append(field3)
+
+        prefixes: List[str] = []
+        prefixes.append("blog:post:")
+
+        """
+        Create an index with multiple fields with Hash data type.
+        """
```

Comment:
`"""` used for docs only
```suggestion
        # Create an index with multiple fields with Hash data type.
```

Reply:
Updated. All comments in test file.